docs/release.md

```diff
@@ -25,31 +25,32 @@ It is possible that a core approver isn't a contrib approver. In that case, the
 
 ## Releasing opentelemetry-collector
 
-1. Update Contrib to use the latest in development version of Core. Run `make update-otel` in Contrib root directory and if it results in any changes submit a draft PR to Contrib. Ensure the CI passes before proceeding. This is to ensure that the latest core does not break contrib in any way. We’ll update it once more to the final release number later.
+1. Update Contrib to use the latest in development version of Core by running `make update-otel` in Contrib root directory. This is to ensure that the latest core does not break contrib in any way. If it results in any changes, submit a PR to Contrib.
+   -  🛑 **Do not move forward until this PR is merged.** �
```

Comment:
```suggestion
   -  🛑 **Do not move forward until this PR is merged.** 
```

Reply:
For some reason I cannot commit this or push an update to it